exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,7 @@
 // Write your function here
-
-var greeting = createGreeting("Daniel");
+function createGreeting(myName){
+    return myName;
+}
+var greeting = createGreeting("Hello, my name is Daniel");
```

Comment:
Different approach here Ryno, I would have put the message in the function and use the parameter there so it is more reusable.